source/_integrations/qrcode.markdown

```diff
@@ -39,3 +39,21 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+## Basic example
+
+An automation using this integration could look like this:
+
+```yaml
+automation:
+  - alias: 'Catch QR code and arm the alarm system'
+    trigger:
+      platform: state
```

Comment:
This needs to be a list, see our YAML style guide: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

```suggestion
      - platform: state
```